torch/csrc/lazy/core/tensor_util.cpp

```diff
@@ -60,6 +60,9 @@ std::vector<BackendDataPtr> CreateTensorsData(
 bool IsSpecialScalar(const at::Scalar& value) {
   if (FLAGS_torch_lazy_handle_special_scalars &&
```

Comment:
should this be:
if ((FLAGS_torch_lazy_handle_special_scalars || FLAGS_torch_lazy_all_numbers_special_scalars) &&

Reply:
They can just both be set to `true`